### PR TITLE
(#5489) Migrate groovy => yaml for ./e2e/yaks

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

File: e2e/yaks/common/kamelet-binding-http/timer-source-binding-display.yaml [hunk 1]
```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 # ---------------------------------------------------------------------------
 
-apiVersion: camel.apache.org/v1alpha1
-kind: KameletBinding
```
Comment (squakez) [inline e2e/yaks/common/kamelet-binding-http/timer-source-binding-display.yaml:19]: Is this the only KameletBinding we still have? we have an open issue to remove them all https://github.com/apache/camel-k/issues/5490 - feel free to tackle into this same PR if you want.

**Review by squakez — APPROVED** [review]

**Comment by tdiesler** [comment]
Fails unrelated in `TestTektonLikeBehavior`

**Comment by squakez** [comment]
Is this good to merge or are you going to include here the Pipe replacements by any chance?

**Comment by tdiesler** [comment]
There are 40 occurrences of KameletBinding in e2e - I'd prefer to sort that out in another PR

**Pull request merged**